[rospy.client][INFO] 2021-09-19 11:46:51,366: init_node, name[/server_endpoint], pid[15215]
[xmlrpc][INFO] 2021-09-19 11:46:51,366: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2021-09-19 11:46:51,366: Started XML-RPC server [http://wng:34637/]
[rospy.impl.masterslave][INFO] 2021-09-19 11:46:51,367: _ready: http://wng:34637/
[xmlrpc][INFO] 2021-09-19 11:46:51,367: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2021-09-19 11:46:51,367: Registering with master node http://localhost:11311
[rospy.init][INFO] 2021-09-19 11:46:51,367: ROS Slave URI: [http://wng:34637/]
[rospy.init][INFO] 2021-09-19 11:46:51,367: registered with master
[rospy.rosout][INFO] 2021-09-19 11:46:51,367: initializing /rosout core topic
[rospy.rosout][INFO] 2021-09-19 11:46:51,368: connected to core topic /rosout
[rospy.simtime][INFO] 2021-09-19 11:46:51,369: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2021-09-19 11:46:51,381: Starting server on 192.168.1.146:10000
[rospy.internal][INFO] 2021-09-19 11:46:51,622: topic[/rosout] adding connection to [/rosout], count 0
[rosout][INFO] 2021-09-19 11:47:21,458: Connection from 192.168.1.146
[rosout][INFO] 2021-09-19 11:47:21,464: RegisterPublisher(execution_check, <class 'panda_move.msg._ExecutionCheck.ExecutionCheck'>) OK
[rospy.internal][INFO] 2021-09-19 11:47:21,509: topic[/execution_check] adding connection to [http://wng:34637/], count 0
[rospy.internal][INFO] 2021-09-19 11:47:21,511: topic[/execution_check] adding connection to [/server_endpoint], count 0
[rospy.core][INFO] 2021-09-19 12:33:17,323: signal_shutdown [signal-2]
[rospy.internal][INFO] 2021-09-19 12:33:17,326: topic[/execution_check] removing connection to /server_endpoint
[rospy.internal][INFO] 2021-09-19 12:33:17,327: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2021-09-19 12:33:17,327: topic[/execution_check] removing connection to http://wng:34637/
[rospy.impl.masterslave][INFO] 2021-09-19 12:33:17,327: signal-2
[rospy.internal][ERROR] 2021-09-19 12:33:17,327: Unable to initiate TCP/IP socket to wng:44619 (http://wng:34637/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 600, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.core][INFO] 2021-09-19 12:33:17,459: signal_shutdown [atexit]
